src/Label/Commands/AbstractCommand.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace CultuurNet\UDB3\Label\Commands;
+
+use ValueObjects\Identity\UUID;
+
+abstract class AbstractCommand
+{
+    /**
+     * @var UUID
+     */
+    private $uuid;
+
+    /**
+     * AbstractCommand constructor.
+     * @param UUID $uuid
+     */
+    public function __construct(UUID $uuid)
+    {
+        $this->uuid = $uuid;
+    }
+
+    /**
+     * @return UUID
+     */
+    public function getUuid()
```

Comment:
Detail, maar volgens mij hebben we vroeger met een getItemId functie gewerkt (zie offer namespace). Is dat hier ook niet genoeg? Of maakt het niet uit? Leek me gewoon beter naar consistentie in udb3 code.